pom.xml

```
@@ -0,0 +1,134 @@
+        </repository>
+    </repositories>
+    <distributionManagement>
+        <repository>
+            <id>slingrRepo.write.release</id>
+            <url>gs://repo.slingrs.io/release</url>
+        </repository>
+        <snapshotRepository>
+            <id>clojars</id>
+            <name>Clojars repository</name>
+            <url>https://clojars.org/repo</url>
+        </snapshotRepository>
+    </distributionManagement>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${project.build.compiler.version}</version>
+                <configuration>
+                    <source>${jdk.version}</source>
+                    <target>${jdk.version}</target>
+                    <encoding>${project.build.sourceEncoding}</encoding>
+                </configuration>
+            </plugin>
+            <!-- This plugin is used to generate the JAR with all the dependencies, which is needed
+            when deploying the service in the platform (not development mode). -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>${project.build.shade.version}</version>
+                <configuration>
+                    <shadedArtifactAttached>true</shadedArtifactAttached>
+                    <shadedClassifierName>jar-with-dependencies</shadedClassifierName>
+                    <transformers>
+                        <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
+                        <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                            <manifestEntries>
+                                <Main-Class>${build.main-class}</Main-Class>
+                                <X-Compile-Source-JDK>${jdk.version}</X-Compile-Source-JDK>
+                                <X-Compile-Target-JDK>${jdk.version}</X-Compile-Target-JDK>
+                            </manifestEntries>
+                        </transformer>
+                        <transformer implementation="org.apache.maven.plugins.shade.resource.ApacheNoticeResourceTransformer">
+                            <addHeader>false</addHeader>
+                        </transformer>
+                        <transformer implementation="org.apache.maven.plugins.shade.resource.ApacheLicenseResourceTransformer"/>
+                    </transformers>
+                    <filters>
+                        <filter>
+                            <artifact>*:*</artifact>
+                            <excludes>
+                                <exclude>META-INF/services/org/apache/camel/component.properties</exclude>
+                                <exclude>META-INF/services/org/apache/camel/dataformat.properties</exclude>
```

One more think related to deployment in cloud
```suggestion
                                <exclude>META-INF/services/org/apache/camel/dataformat.properties</exclude>
                                <exclude>META-INF/*.SF</exclude>
                                <exclude>META-INF/*.DSA</exclude>
                                <exclude>META-INF/*.RSA</exclude>
```

---

Make sure of copy all the content of this file `pom.xml` to `public_pom.xml` the extensions-builder use the public file